multisig-transactions-monitor/src/utils.js

```diff
@@ -0,0 +1,470 @@
+/* eslint-disable global-require */
+const { Finding, FindingType, FindingSeverity } = require('forta-agent');
+
+const config = require('../agent-config.json');
+
+const multisigAddress = config.contracts.multisig.address;
+
+/// CREATE FINDINGS FOR GNOSIS SAFE INTERACTIONS ///
+/*
+        "AddedOwner",
+        "RemovedOwner",
+*/
+function createAddOwnerFinding(
+  log,
+  protocolName,
+  protocolAbbreviation,
+  developerAbbreviation,
+) {
+  const { owner } = log.args;
+
+  const finding = Finding.fromObject({
+    name: `${protocolName} Multisig Owner Added`,
+    description: `Address ${owner} was added as an owner`,
+    alertId: `${developerAbbreviation}-${protocolAbbreviation}-MULTISIG-OWNER-ADDED-ALERT`,
+    protocol: config.protocolName,
```

Comment:
Because you are already passing in the `protocolName` variable to all of these functions, you can remove the import of the `agent-config.json` file and update all of the `protocol:` values to use the function argument.
```suggestion
    protocol: protocolName,
```